docs/tutorial/creating-new-tools.md

```diff
@@ -0,0 +1,154 @@
+
+This page describes the process of creating a Geneweaver Analysis Tool that can be 
+integrated directly with rest of the Geneweaver ecosystem, including the Geneweaver
+web application.
+
+To create a new tools, you will need to write the interface for your tool in Python,
+and then write the interface for your tool in JavaScript. The Python interface will
+be used to run your tool locally, and the JavaScript interface will be used to run
+your tool in the web application.
+
+Beneath the python interface, you can use any language you want to implement your
```

Comment:
Maybe add this sentence to the paragraph above ending on line 9. That way we have a comprehensive understanding of necessary languages. Start this paragraph with "Your tool must have a public interface implemented in Python and be available as a docker image. Below, we will describe how to ...."